src/common/darktable.c

```diff
@@ -78,7 +78,9 @@
 #include <sys/wait.h>
 #endif
 #include <locale.h>
+#if defined(__SSE__)
```

Comment:
Well, i'm sure they can, but we will still need to place `#if defined(__SSE__)` around `process_sse2()` (and similar) functions, so i do not see any point in that..